docs/change_controlled.rst

```diff
@@ -0,0 +1,82 @@
+.. _LDM-294: https://ls.st/LDM-294
+
+.. note::
+  The definition of which types of documents should be change-controlled is located in `LDM-294`_.
+
+
+Developer Workflow
+==================
+
+The development of change-controlled documents differs from :ref:`that of code <git-branching>`, in that some documents under development can take a long time to evolve and can be developed under multiple stories (ticket branches), and by multiple authors.
+Repositories associated with change-controlled documents should therefore use two protected branches: ``master`` should correspond solely to an archived and approved instance of the document on DocuShare, and ``draft`` is a long-lived integration branch that can be used as the document evolves.
+
+There can be cases where some minor fixes to a controlled document must be made whilst more major changes are being developed using the ``draft`` branch.
+It is therefore allowed for ticket branches to be created off of ``master`` if the changes are shortly going to be sent for CCB approval.
+This is the usual workflow for controlled documents generated from the SysML model where the content is developed in an external tool and only exported to latex periodically.
+
+Since neither ``draft`` nor ``master`` can be rewritten, it is important that if branches are created off of ``master`` and the content is still present in ``draft``, that ``master`` be merged back into the ``draft`` branch to ensure that ``draft`` is using a consistent version of the baseline.
+Given this requirement, it is preferred that ``draft`` be used in the usual case.
+Regardless of which base branch is used, ticket branches should follow standard :ref:`DM branching policies <git-branching>`.
+
+Once a document handle has been issued a repository can be created in the ``lsst`` GitHub organization using a repository name that matches the handle.
+If the document exists in a pre-existing repository, it should be renamed to the handle and, if necessary, moved to the correct organization.
+
+.. warning::
+  It may be necessary to adjust the history of a pre-existing repository if it has been developed using different policies, in particular it is important that ``master`` be adjusted such that it only contains merge commits from approved versions (and the initial placeholder commit).
+
+All documents use a Travis configuration file to build the document and deploy it to the corresponding https://ldm-nnn.lsst.io/ web site.
+Built PDFs of documents should not be committed to the repository.
+They are deployed automatically to the web during development and the final version will also be archived on DocuShare.
+
+During development:
+
+* Use the ``lsstdraft`` class option to make it clear that the document has not been approved.
+* Specify a date using :code:`\today` so that the built document continues to have an updated date for each change (we do not automatically insert a date based on the state of the git repository).
```

Comment:
Shouldn't we do this based on git? Naïvely, it seems like the obviously-correct approach.